src/plugins/controls/public/control_types/range_slider/range_slider.scss

```diff
@@ -0,0 +1,57 @@
+.rangeSlider__popoverOverride {
+  height: 100%;
+  max-width: 100%;
+  width: 100%;
+}
+
+@include euiBreakpoint('m', 'l', 'xl') {
+  .rangeSlider__panelOverride {
+    min-width: $euiSizeXXL * 12;
+  }
+}
```

Comment:
@cqliu1 was just using the Cloud environment to run through the questions for the usability testing sessions and I came across this bug. The range popover panel gets cut off when it is on the right side.
<img width="1534" alt="image" src="https://user-images.githubusercontent.com/4016496/160330195-f2d91ff0-fb7d-4511-8a94-a290aea76658.png">

Removing this media query should solve the issue.

<img width="532" alt="image" src="https://user-images.githubusercontent.com/4016496/160331703-4b0b8c8d-aa0b-469a-9c5a-42c7eca4e16c.png">

Reply:
@cqliu1 But we would need to figure out how to handle the small controls after removing that media query
<img width="249" alt="image" src="https://user-images.githubusercontent.com/4016496/160331954-bbcd4dce-1647-4030-afd0-3c4c08a216b9.png">

Reply:
The quickest suggestion I can think of is to increase the size of the [small](https://github.com/elastic/kibana/blob/ca81ce870710c1ec31f2bb6736ce426be95b90b4/src/plugins/controls/public/control_group/control_group.scss#L1) controls to at least 300px (`$euiSizeM * 25`). What do you think @ThomThomson ?

<img width="261" alt="image" src="https://user-images.githubusercontent.com/4016496/160332703-7bfb585e-cc46-4cb4-ae02-8e240127e8cd.png">

Reply:
And likely using `EuiResizeObserver` is a more appropriate fix (like Option List is doing).

Reply:
The reason why I'm using `EuiInputPopover` is bc there were focus issues with using the `EuiPopover` + `EuiResizeObserver` where you would click on the lower bound number field of the anchor but focus would jump to the range slider in the popover, but if that would solve the popover sizing issues, then it might be worth finding a fix for the focus stuff. I agree with Devon and can address follow up PR.

Reply:
++ on fixing in a follow-up PR.